pkg/helper/rolepolicybindings/helpers.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rolepolicybindings
+
+import (
+	"fmt"
+	"strings"
+
+	"slices"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+	"github.com/openshift/rosa/pkg/aws"
+)
+
+const (
+	RolePolicyBindingFailedStatus = "failed"
+)
+
+func ValidateRolePolicyBindings(bindings *cmv1.RolePolicyBindingList) error {
```

Comment:
NP : update the naming of this func to better describe what we are doing here, we are only checking the bindings   status values, we are not doing any other validations IMO

Reply:
change the function name to CheckRolePolicyBindingStatus()